intro-HPC/ch_running_batch_jobs.tex

```diff
@@ -353,8 +353,8 @@ \subsection{Search for modules}
 %\ifgent{}%
     You will need to load all module(s) on any one of the lines below before the "example/1.2.3" module is available to load.
 
-      cluster/golett
-      cluster/phanpy
+      cluster/kirlia
+      cluster/joltik
```

Comment:
doduo?

Reply:
ah, you want it exact, I thought they were just examples

Reply:
It is just an example, but we usually have all software installed on all clusters, so they should be all listed here (except the hidden ones).

```
      cluster/doduo
      cluster/joltik
      cluster/kirlia
      cluster/skitty
      cluster/swalot
      cluster/victini
```